downloadFlowEntities.sh

```diff
@@ -55,15 +71,20 @@ function downloadZipFile {
         if [ -r "${outputFile}" ] ; then
             unzip "${outputFile}" -d "./src/main/${outputDirectory}/" > /dev/null || true
             rm "${outputFile}"
+            # Use jq to format the JSON files that were in the zip file.
+            while IFS= read -r -d '' jsonFile ; do
+                cat <<< "$(jq < "${jsonFile}")" > "${jsonFile}"
+            done < <(find "./src/main/${outputDirectory}" -iname '*.json' -print0)
         fi
    fi
 }
 
-#                Flow Route            Directory (if different)
-#                -------------------   --------------------------
-downloadJsonFile flows
-downloadJsonFile sharedConfig
+#                Flow Route            Directory        Query String Parameters
+#                -------------------   ------------     ---------------------------
+downloadJsonFile flows                 flows
+downloadJsonFile sharedConfig          sharedConfig     '?encrypted=true'
```

Comment:
please use `encrypted_only=true` here instead

Reply:
Resolved in: https://github.com/ucroo/ihub-developer-scripts/pull/17